vllm/model_executor/layers/quantization/awq_hpu.py

```diff
@@ -0,0 +1,222 @@
+from typing import Any, Dict, List, Optional
```

Comment:
Shouldnt it be added to vllm-extension?

Reply:
CUDA class file https://github.com/HabanaAI/vllm-fork/blob/f2fd3586394a3b08fc5d26164a702d0fd4e1d6e7/vllm/model_executor/layers/quantization/awq.py also has these imports

Reply:
Please move this class to vllm-hpu=extension repo

Reply:
@michalkuligowski, I moved the class to extension.